login.js

```diff
@@ -32,17 +71,31 @@ async function pushUserObject() {
   } else {
     throw new Error(`invalid status code: ${res.status}`);
   }
-}
+} */
 function updateUserObject() {
-  const loginEmailStatic = document.getElementById('login-email-static');
   const userName = document.getElementById('user-name');
   // const avatarName = document.getElementById('avatar-name');
-  loginEmailStatic.innerText = userObject.name;
   userName.innerText = userObject.name;
   // avatarName.innerText = userObject.avatarHash !== null ? userObject.avatarHash : 'None';
 
   loginManager.pushUpdate();
 }
+async function finishLogin(newLoginToken) {
+  await storage.set('loginToken', newLoginToken);
+
+  loginToken = newLoginToken;
+
+  console.log('got user token', loginToken);
+
+  const loginForm = document.getElementById('login-form');
+  // document.body.classList.add('logged-in');
+  loginForm.classList.remove('phase-1');
```

Comment:
To do: not a huge fan of the phase system. It works tho for now. Preact can handle this with some simple props now.

Reply:
I agree, though that is not related to this PR.